src/components/Layout/Header/Header.tsx

```diff
@@ -55,7 +64,23 @@ export const Header = () => {
         zIndex='banner'
         top={0}
       >
-        <HStack width='full' px={4}>
+        {walletInfo?.deviceId === 'DemoWallet' && (
+          <Box
+            bg='blue.500'
+            width='full'
+            height='2.5rem'
+            fontSize={{ base: 'sm', md: 'md' }}
+            as='button'
+            onClick={handleBannerClick}
+          >
+            <HStack verticalAlign='middle' justifyContent='center' spacing={3}>
+              <InfoIcon boxSize='1.3em' color='white' />
+              <Text color='white' fontWeight='bold' translation='navBar.demoMode' />
+              <Text color='white' fontWeight='medium' translation='navBar.clickToConnect' />
+            </HStack>
+          </Box>
+        )}
+        <HStack height='calc(4.5rem - 1px)' width='full' px={4}>
```

Comment:
What's that `calc` for?

Reply:
This was to address a 1px scroll height bug with the SideNav component.
However, I have moved the bottom border to the last child in the header, instead of the Flex wrapper, so this 1px offset is no longer required and has been removed.